src/Config.php

```diff
@@ -1601,6 +1603,10 @@ public static function getAllConfigData()
         }
 
         include $configFile;
+        if (false === isset($phpCodeSnifferConfig)) {
+            $phpCodeSnifferConfig = null;
```

Comment:
The simpler way to do this is to turn the order around:
```php
$phpCodeSnifferConfig = null;
include $configFile;
```
... has the same result with less logic involved.